TypeDBConsole.java

```diff
@@ -771,7 +772,7 @@ private static class CLIOptions implements Runnable {
         @CommandLine.Option(
                 names = {"--password"},
                 description = "Password",
-                prompt = "Enter password:",
+                prompt = "Password: ",
```

Comment:
What is the use of `prompt` if we're reading the password programmatically in the code with `Utils.readPassword(passwordReader, "Password: ");`?

Reply:
This is specifically for the launching of Console from a shell and connecting to Cluster, not anything within console's REPL. The experience would look something like:
```
$ ./typedb console --cluster=localhost:1729 --username admin 
Password: ********
> transaction schema write [...]
```

Reply:
Oh right! Got it.